src/ducks/reimbursements/HealthReimbursements.jsx

```diff
@@ -2,20 +2,58 @@ import React, { Component } from 'react'
 import { connect } from 'react-redux'
 import { queryConnect } from 'cozy-client'
 import { transactionsConn } from 'doctypes'
-import { flowRight as compose } from 'lodash'
+import { flowRight as compose, sumBy } from 'lodash'
 import { getHealthExpensesByPeriod } from 'ducks/filters'
 import { TransactionsWithSelection } from 'ducks/transactions/Transactions'
 import withBrands from 'ducks/brandDictionary/withBrands'
 import withAppsUrls from 'ducks/apps/withAppsUrls'
+import { isFullyReimbursed } from 'ducks/transactions/helpers'
+import { translate } from 'cozy-ui/react'
+import { Title } from 'cozy-ui/react/Text'
+import { Padded } from 'components/Spacing'
+import { Figure } from 'components/Figure'
+import styles from 'ducks/reimbursements/HealthReimbursements.styl'
 
 class RawHealthReimbursements extends Component {
   render() {
+    const { filteredTransactions, t } = this.props
+
+    const reimbursedTransactions = filteredTransactions.filter(
+      isFullyReimbursed
+    )
+
+    const awaitingTransactions = filteredTransactions.filter(
+      t => !isFullyReimbursed(t)
+    )
```

Comment:
const { true: reimbursedTransactions, false: awaitingTransactions } = groupBy(filteredTransactions, isFullyReimbursed)

IMHO this should be in redux.

Reply:
Thanks. OK for putting this in Redux, but not now since I'm not totally comfortable this that and we have a deadline.